src/v3/src/transformer/transaction/transform.ts

```diff
@@ -13,26 +13,37 @@
 import { APIError, IdxRemediation } from '@okta/okta-auth-js';
 import { flow } from 'lodash';
 
-import { IDX_STEP } from '../../constants';
+import { AUTHENTICATOR_KEY, IDX_STEP } from '../../constants';
 import { RegistrationElementSchema, TransformStepFnWithOptions } from '../../types';
 import {
   convertIdxInputsToRegistrationSchema,
   convertRegistrationSchemaToIdxInputs,
+  getAuthenticatorKey,
   parseRegistrationSchema,
   triggerRegistrationErrorMessages,
 } from '../../util';
 
 export const PIV_TYPE = 'X509';
 
 const transformRemediationNameForIdp: TransformStepFnWithOptions = (options) => (formbag) => {
-  const { transaction: { neededToProceed: remediations, nextStep } } = options;
+  const { transaction } = options;
+  const { neededToProceed: remediations, nextStep, context } = transaction;
   if (!remediations.length) {
     return formbag;
   }
 
+  if (nextStep?.name === IDX_STEP.REDIRECT_IDP
+    && getAuthenticatorKey(transaction) === AUTHENTICATOR_KEY.IDP) {
+    // idp authenticator
+    const isVerifyFlow = Object.prototype.hasOwnProperty.call(context, 'currentAuthenticatorEnrollment');
```

Comment:
I think you can use es6 for this 

```
// currentAuthenticator is from enrollment flows and currentAuthenticatorEnrollment is from verify flows
const { rawIdxState: { currentAuthenticator, currentAuthenticatorEnrollment } } = transaction;
const isVerifyFlow = typeof currentAuthenticatorEnrollment !== 'undefined';
```

To make this re-usable, you can create it as a util function.